.github/workflows/tests.yaml

```diff
@@ -0,0 +1,101 @@
+name: tests
+
+on:
+  push:
+  pull_request:
+  schedule:
+    - cron: "0 0 * * *"
```

Comment:
Could we not use cron crons please? We wouldn't be using any external dependencies, so I don't think we have to test on a schedule.